tests/debian_tests.sh

```diff
@@ -5,14 +5,17 @@ sudo cp jwt.htpasswd /var/www/jwt.htpasswd
 sudo mkdir -p /var/www/testjwt/
 sudo touch /var/www/testjwt/index.html
 
-mkdir -p /opt/mod_jwt_tests
+sudo mkdir -p /opt/mod_jwt_tests
 
 sudo openssl ecparam -name secp256k1 -genkey -noout -out /opt/mod_jwt_tests/ec-priv.pem
 sudo openssl ec -in /opt/mod_jwt_tests/ec-priv.pem -pubout -out /opt/mod_jwt_tests/ec-pub.pem
 
 sudo openssl genpkey -algorithm RSA -out /opt/mod_jwt_tests/rsa-priv.pem -pkeyopt rsa_keygen_bits:4096
 sudo openssl rsa -pubout -in /opt/mod_jwt_tests/rsa-priv.pem -out /opt/mod_jwt_tests/rsa-pub.pem
 
+sudo chmod 644 /opt/mod_jwt_tests/*.pem
```

Comment:
apache and the python weren't able to read the private keys so were giving errors when running the tests